src/Tribe/Views/V2/Template.php

```diff
@@ -59,8 +59,12 @@ class Template extends Base_Template {
 	public function render( array $context_overrides = [] ) {
 		$context = wp_parse_args( $context_overrides, $this->context );
 		$context['_context'] = $context;
+		$template_slug = array_filter( [
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: d3c9c541141ff3544e53e182d8c7add1</sub>